src/Kind.hs

```diff
@@ -0,0 +1,10 @@
+module Kind where
+
+
+data Kind
+  = KStar
+  | KFun Kind Kind
+  deriving (Show, Eq)
```

Comment:
if we do kind inference, eventually this will have to be split into a non-recursive type, like everything else.

Reply:
Indeed!

Reply:
Implemented in #92.